web/styles/settings.css

```diff
@@ -1893,6 +1893,13 @@ $option_title_width: 180px;
     #change_email_modal {
         width: 300px;
     }
+
+    .active,
+    .user_name_column {
+        position: sticky;
+        z-index: 2 !important;
```

Comment:
Here also you can avoid using `important` by using a more specific selector.

Reply:
Same reason as above.